recipes/qwt/all/test_package/example.cpp

```diff
@@ -0,0 +1,7 @@
+#include <qwt_plot.h>
+
+int main()
+{
+    QwtPlot* plot;
+    return 0;
```

Comment:
You should also call a function of qwt to test that linking actually works, because this test can pass only with an include path properly set-up, but no library.
Something likde `qDebug() << QwtDate::toString(QDateTime::now());` for example (I did not test it)